agent/src/config/fileconfigloader.cpp

```diff
@@ -17,6 +18,8 @@
  * For further information see LICENSE
  */
 
+
+#include "softwarecontainer-common.h"
```

Comment:
This include should be in `fileconfigloader.h` instead of the cpp file, since you declare a log context there.